### ISSUE TITLE
A11y: Text of the light button with too little contrast

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7041]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The text of the light button (CSS selector button.btn-light) on narrower viewports has too low a contrast ratio between grey text and grey background color.

This can be difficult or impossible for people with a visual impairment to perceive.

*_Expected behaviour:_*

The contrast ratio must be increased to at least 4.5:1.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. open a menu (e.g. offcanvas cart)

![Image](https://github.com/user-attachments/assets/649f32bd-dfe3-4392-9f8d-a844909410be)

![Image](https://github.com/user-attachments/assets/e1694017-476f-4030-a525-8afd775037a1)

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40813**



[created from NEXT-40813, comment 541358]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40813, comment 541559]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.3**

 This issue will be / was solved in Shopware version ***6.6.10.3***.



[created from NEXT-40813, comment 544066]
